src/components/registration-lite.js

```diff
@@ -136,6 +136,12 @@ const RegistrationLite = (
         loadProfileData,
         closeWidget,
         hasVirtualAccessLevel,
+        initialOrderComplete1stParagraph,
```

Comment:
this is not correct
please strategy used here
https://github.com/fntechgit/summit-registration-lite/blob/2d8a39f6f09729c69131c526732ccc6bff598fcc/src/components/registration-lite.js#L372